go.mod

```diff
@@ -224,4 +228,4 @@ require (
 )
 
 // the replacement build is based on https://github.com/dragonchaser/cs3apis/tree/master
-replace github.com/cs3org/go-cs3apis => github.com/dragonchaser/go-cs3apis v0.0.0-20230918130959-ae732d4b8147
+replace github.com/cs3org/go-cs3apis => github.com/aduffeck/go-cs3apis v0.0.0-20231009082215-ad45e19edac0
```

Comment:
so this includes all the fixes we added over time, right?

Reply:
Yes. Actually, all our changes have been merged back into master by now except for the hidden shares one which is part of my branch. See https://github.com/cs3org/go-cs3apis/compare/main...aduffeck:go-cs3apis:main